src/sql/ggplot/facet_wrap.py

```diff
@@ -21,6 +21,9 @@ def get_facet_values(self, table, column, with_):
         conn = sql.connection.Connection.current
 
         values = conn.execute(query, with_).fetchall()
+        # Filter out None values
+        # Added to make histogram more inclusive to NULLs
+        values = [value for value in values if value[0] is not None]
```

Comment:
What does value[0] define?

Reply:
`values = conn.execute(query, with_).fetchall()` returns all the unique values in the facet that we use for ggplot. 
Added this line `values = [value for value in values if value[0] is not None]` to remove  NULL  from the list of unique values in the facet. This is so that we don't generate a plot for the NULL value.

Reply:
Can u add a comment here that value[0] is the null value.